doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1118,6 +1118,16 @@ KRKNWK-11666: CLI command ``hap services`` returns incorrect results
 KRKNWK-11365: HAP tool's ``provision`` command freezes
   This issue happens on macOS when an EUI argument is not passed in attempt to read EUI from a connected board.
 
+Google Fast Pair Service
+========================
+
+The issues in this section are related to the :ref:`bt_fast_pair_readme`.
+
+.. rst-class:: v2-5-0
```

Comment:
I think you shouldn't add here future releases, only the past ones. I guess the issue is present in the v2.4.0 release. It would be also great to check if previous releases are also affected

(v2.5.0 tag will be added to this issue during the v2.5 release process)

Reply:
Previous releases are not affected with this one (the issue was introduced together with fix for possibility of unauthenticated CCC overwrite ...)

Reply:
I see. I wasn't aware of it